### PR TITLE
feat: triggers

### PR DESCRIPTION
**Pull request by nduchak**

### CODE REVIEW

File: deploy.sh [hunk 1]
```diff
@@ -1,25 +1,25 @@
 #!/bin/sh
 
 if [[ $# -eq 0 ]] ; then
-    echo 'Please specify the services to deploy. Services available: "rns" "storage".'
+    echo 'Please specify the services to deploy. Services available: "rns" "storage" "triggers".'
```
Comment (julianmrodri) [inline deploy.sh:4]: I would rename the whole service and folder to `notifications` just to be consistent with UI and how we name the services in the other parts of the Marketplace.

**Review by julianmrodri — CHANGES_REQUESTED** [review]
Awesome! Looks really good but just think we should rename `triggers` to `notifications` for consistency. Its kind of a pain because you need to replace everywhere but its the only change, the rest looks perfect.

File: deploy.sh (hunk 1 above)
Comment (nduchak) [reply deploy.sh]: Done

File: README.md [hunk 2]
```diff
@@ -287,4 +287,6 @@ sh forward.sh
 # Troubleshooting
 ### RNS manager missmatch between networks
 Solution: switch back and forth a network on MetaMask/Nifty. If that does not work make sure you have setup correctly the network id in the RNS step.
-    
\ No newline at end of file
+    
+    
+    https://github.com/rsksmart/rif-notifier/tree/rgwy135
```
Comment (julianmrodri) [inline README.md:292]: lets remove this line and we can do the Trigger running in separate PR.

**Review by julianmrodri — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (nduchak) [reply README.md:292]: done
